Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/helper.go

```diff
@@ -320,6 +320,11 @@ func NewOrDie(c *Config) *Client {
 // running inside a pod running on kuberenetes. It will return an error if
 // called from a process not running in a kubernetes environment.
 func InClusterConfig() (*Config, error) {
+	host, port := os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")
```

Comment:
I acknowledge that this was pre-existing, but seems out of keeping.  The rest of the config is coming in as secrets, we should probably just have a complete kubeconfig available to mount.